internal/cmd/configssh.go

```diff
@@ -208,7 +209,7 @@ func makeNewConfigs(binPath, userName string, workspaces []coderutil.WorkspaceWi
 
 func makeSSHConfig(binPath, host, userName, workspaceName, privateKeyFilepath string, tunnel bool) string {
 	if tunnel {
-		return fmt.Sprintf(
+		host := fmt.Sprintf(
```

Comment:
Is it possible/straightforward to add an integration test for this?

Reply:
Good idea.... will do!